### ISSUE TITLE
Keyboard access?

### ISSUE THREAD

**Issue by StephenMeehan2016** [greghub/green-audio-player#19]
Hi, 

I was wondering, would it be possible to add keyboard support to the play/pause button?

I've added `tabindex="0"` to `class="play-pause-btn"` this includes it in the tab sequence, but pressing return doesn't play/pause the audio.

Could you also add `aria-pressed="false"` and `aria-pressed="true"` if `class="play-pause-btn"`has been pressed or not.

This would improve the accessibility of Green audio player.

**Comment by tnbnicer** [comment]
I believe what you may be looking for is in the fork I made [here](https://github.com/tnbnicer/green-audio-player). The player has accessibility options, keystrokes for pause/play, volume and time-seek, and aria. Keyboard support is difficult (not impossible) to implement with multiple players on one page.

You should download the fork and try out the example "examples/single-instance-2.html", to see if it does what you like.

**Comment by StephenMeehan2016** [comment]
Thanks for responding, I'll take look. Is there anything special I need to do to make it work with multiple players on one page?

**Comment by tnbnicer** [comment]
Probably iframes and a single-instance player in each frame would be your best chance of getting it to work. The problem is essentially focus. To disable the keystrokes for a player instance the focus has to be on another player. In addition to stopping a player when another player starts, you need to change the focus between players, That would be the easy part. The hard part is keeping the focus on a player while it's playing. I guess.

**Comment by tnbnicer** [comment]
I hope this does not come too late for you.

I added tabindexes in my [fork](https://github.com/tnbnicer/green-audio-player), mentioned above. It is currently a pull request, so it could become part of the main player branch. Before I only implemented keystrokes without tabindexes.

You enable **tabindex** accessibility as follows:

`new GreenAudioPlayer('.ready-player-1', { enableKeystrokes: true });`

Or, using init options:

```
GreenAudioPlayer.init({
    selector: '.player-with-accessibility',
    stopOthersOnPlay: true,
    enableKeystrokes: true
});
```
See the demos (in the fork):
../examples/multiple-instance.html
../examples/single-instance.html

Lastly, [here's a link](https://www.teanow5pm.co.uk/midiifrap.php) to a live player with different audio choices. The blue theme player is `mediaelement`. The green theme one is `green-audio-player`. Both support tabindexes.
